lib/rules/web/admin_console/4.8/logs.xyaml

```diff
@@ -106,5 +106,31 @@ toggle_log_wraps:
     selector:
       xpath: //input[@id='wrapLogLines']
     op: click
+check_logs_history_not_present:
+  elements:
+  - selector:
+      xpath: //button[@disabled]//span[text()='Current log']
+  - selector:
+      xpath: //div[@class='pf-c-select']
+    op: hover
+  - selector:
+      xpath: //div[@class='pf-c-tooltip__content' and contains(text(),'<message>')]
+check_logs_history_is_present:
+  elements:
+  - selector:
+      xpath: //button//span[text()='<log_current>']
+  - selector:
+      xpath: //div[@class='pf-c-select']
+    op: click
+  - selector:
+      xpath: //button[@class='pf-c-select__menu-item pf-m-selected' and contains(text(),'<log_current>')]
+  - selector:
+      xpath: //button[@class='pf-c-select__menu-item' and contains(text(),'<log_previous>')]
```

Comment:
same as above

Reply:
see combined comment for all class text matching comment below